src/views/AddItem.jsx

```diff
@@ -25,18 +26,25 @@ export function AddItem() {
 		setFormData((formData) => ({ ...formData, [name]: value }));
 	};
 
+	const itemNames = data.map((item) =>
+		item.name.toLowerCase().replace(/[^a-z0-9]/gi, ''),
```

Comment:
Nice use of regex!

Reply:
The replace method is very powerful combined with the regex! Lots of ways to do the matching! This is as good as any `[^a-z0-9]` 🎉